node_modules/oae-core/documentpreview/js/documentpreview.js

```diff
@@ -108,14 +108,14 @@ define(['jquery', 'underscore', 'oae.core', 'lazyload'], function($, _, oae) {
             page.height = page.$el.height();
             page.marginTop = CONTENT_PAGE_SPACING;
 
-            // Request the page content
+            // Request the page content HTML version
             $.ajax({
```

Comment:
Any reason why they have to be combined? This wouldn't work really well with a storage backend such as S3 as the backend servers would have to act as a proxy and would suck up quite a bit of bandwidth/IO.